hdl/testbench/xwb_fofb_processing/Manifest.py

```diff
@@ -0,0 +1,22 @@
+action = "simulation"
+sim_tool = "ghdl"
+sim_top = "xwb_fofb_processing_tb"
+target = "xilinx"
+syn_device = "xc7a200t"
+
+ghdl_opt = "--std=08 -g -frelaxed"
+sim_post_cmd = "ghdl -r xwb_fofb_processing_tb --stop-time=200us --vcd=xwb_fofb_processing_tb.vcd && gtkwave xwb_fofb_processing_tb.vcd"
```

Comment:
With VHDL-2008 you can use the `std.env.finish;` statement at the end on the test process. I find it better than hard-coding the stop time.

Reply:
Use `--wave=xwb_fofb_processing_tb.ghw` instead of `--vcd...` to use a better file format for VHDL simulations (see: [https://ghdl.github.io/ghdl/ghw/index.html](https://ghdl.github.io/ghdl/ghw/index.html))

I would also avoid calling gtkwave at the end without user interaction. This brakes automatic batch tests.